src/pages/developers/index.page.tsx

```diff
@@ -91,11 +91,11 @@ const resourceData = {
       content: 'The Build Club provides support for teams and individuals looking to launch new projects on Nervos.',
       link: 'https://www.buildclub.xyz/',
     },
-    // {
-    //   title: 'CKB Training Course',
-    //   content:
-    //     'Detailed tutorials about creating CKB transactions and details about the structure of the Layer 1 CKB blockchain.',
-    // },
+    {
+      title: 'Nervos Talk Forum',
+      content: 'For the community to participate and discuss developments on Nervos.',
+      link: 'https://talk.nervos.org/',
+    },
```

Comment:
Only the first 3 links are mentioned in https://docs.google.com/document/d/1abU8b1RSSDIXN3IquKbdT7ukxLCoIyaZg8F32-XtJow/ as developer resources

Reply:
This is the designer's comment https://nervos-official-website-git-preview-magickbase.vercel.app/developers#thread-id=Jdgb5